src/app/api/v1/user/remove-email/route.ts

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getToken } from "next-auth/jwt";
+import { NextRequest, NextResponse } from "next/server";
+import AppConstants from "../../../../../appConstants";
+
+import {
+  getSubscriberByEmail,
+  deleteResolutionsWithEmail,
+} from "../../../../../db/tables/subscribers";
+import {
+  removeOneSecondaryEmail,
+  getEmailById,
+} from "../../../../../db/tables/emailAddresses.js";
+import { getL10n } from "../../../../functions/server/l10n";
+
+interface EmailDeleteRequest {
+  emailId: string;
+}
+
+export async function POST(req: NextRequest) {
+  const l10n = getL10n();
+  const token = await getToken({ req });
+
+  if (token) {
+    try {
+      const { emailId }: EmailDeleteRequest = await req.json();
+      const subscriber = await getSubscriberByEmail(token.email);
+      const existingEmail = await getEmailById(emailId);
+
+      if (existingEmail?.subscriber_id !== subscriber.id) {
+        return NextResponse.json({
+          success: false,
+          status: 400,
+          message: l10n.getString("error-not-subscribed"),
+        });
```

Comment:
Q: Should the `status` be part of the response, or a second parameter
```js
        return NextResponse.json(
          {
            success: false,
            message: l10n.getString("error-not-subscribed"),
          },
         { status: 400 }
        );
```

For example:

https://github.com/mozilla/blurts-server/blob/82c28e2f27eca0cfcafad455107be185dd0dd8cc/src/app/api/v1/user/email/route.ts#L91-L97

Reply:
Oh! Nice catch, I'm fairly sure those should be in the second argument: 52cb724cf

/cc @mansaj just in case